easybuild/easyconfigs/c/CrayCCE/CrayCCE-19.08.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayCCE'
+version = '19.08'
+
+homepage = 'https://pubs.cray.com/discover'
+description = """Toolchain using Cray compiler wrapper, using PrgEnv-cray
+(PE release: June 2019).
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}  # (default) version
+
+dependencies = [
+    ('PrgEnv-cray', EXTERNAL_MODULE),
+    ('atp/2.1.3', EXTERNAL_MODULE),
+    ('cce/9.0.1', EXTERNAL_MODULE),
+    ('modules/3.2.11.2', EXTERNAL_MODULE),
```

Comment:
We current have an issue with using non default `modules`, therefore I suggest that you remove this from the dependencies (for the moment we will rely on the default version)

Reply:
Done.